docs/guides/qedma-qesem.ipynb

```diff
@@ -324,22 +332,11 @@
      "text": [
       "The result of the submitted job had 1 PUB\n",
       "The expectation values measured from this PUB are: \n",
-      "[0.98, 1.01]\n",
+      "[0.99946584 0.98926388]\n",
       "The error-bar values are: \n",
-      "[0.028, 0.015]\n",
+      "[0.00337337 0.01438304]\n",
       "And the associated metadata contains the original observables, making the results easier to interpret: \n",
-      "{\n",
-      "    'observables': [\n",
-      "        SparsePauliOp(\n",
-      "            ['IIIIZ', 'IIIZI', 'IIZII', 'IZIII', 'ZIIII'],\n",
-      "            coeffs=[0.2 + 0.0j, 0.2 + 0.0j, 0.2 + 0.0j, 0.2 + 0.0j, 0.2 + 0.0j]\n",
-      "        ),\n",
-      "        SparsePauliOp(\n",
-      "            ['IIIZZ', 'ZIXII'],\n",
-      "            coeffs=[1.0 + 0.0j, 0.5 + 0.0j]\n",
-      "        )\n",
-      "    ]\n",
-      "}\n"
+      "{}\n"
```

Comment:
This looks off; probably because we didn't wait for the job to finish. I'll revert this notebook and make an issue for it.